lucien-ios/APIClient/LucienAPIClient.swift

```diff
@@ -93,4 +93,59 @@ final class LucienAPIClient: APIClient {
             }
         }
     }
+
+    func createPhotoURL(completion: @escaping (Result<AmazonS3ImageURL>) -> Void) {
+        let lucienRequest = LucienRequest.createPhotoURL
+        let urlRequest = lucienRequest.urlRequest
+        sendRequest(urlRequest) { response in
+            switch response {
+            case .success(let result):
+                guard let result = result else {
+                    return completion(.failure(LucienAPIError.noResult))
+                }
+                let decoder = JSONDecoder()
+                decoder.dateDecodingStrategy = .iso8601
```

Comment:
Do you need this here? Do you know what it's doing?

Reply:
I don't need it. That was a mistake.